source/documentation/deploying_services/postgres.md

```diff
@@ -118,9 +118,7 @@ To create a service and bind it to your app:
 
 Your app must make a [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) connection to the PostgreSQL service. Most libraries use TLS by default.
 
-Your app will need to parse the ``VCAP_SERVICES`` [environment variable](/#system-provided-environment-variables) to get details of the PostgreSQL service (or use a library that does so).
-
-(Note that for some languages/frameworks, the Cloud Foundry buildpack will automatically parse ``VCAP_SERVICES`` and set `DATABASE_URL` to the first database found.)
+The Cloud Foundry buildpack will automatically parse the ``VCAP_SERVICES`` [environment variable](/#system-provided-environment-variables) to get details of the PostgreSQL service. The buildpack will set `DATABASE_URL` to the first database found.
```

Comment:
technically it's not buildpack doing anything here, but cloud controller. I'd say something like

CF will automatically parse VCAP services to get details of the service and then set DATABASE_URL variable